### ISSUE TITLE
not working in SFCs

### ISSUE THREAD

**Issue by Bader-Idris** [aaron-bond/better-comments#523]
### this
![image](https://github.com/aaron-bond/better-comments/assets/93044377/26e62d84-6654-4206-8865-2a0e971d452f)
### and this
![image](https://github.com/aaron-bond/better-comments/assets/93044377/27627b65-9086-4b60-b6f7-90a9d62609af)

**Comment by goldengecko** [comment]
Yeah, I wanted to use it to highlight `// #region` and `// #endregion` lines in my vue files, but it would only do the highlighting in plain js/ts files and not Vue files. In Vue files it would highlight in `/** ... */` style comments, which is no use since it needs to be in a `//` style comment for the region stuff to work.

**Comment by edwinhuish** [comment]
try this: https://github.com/edwinhuish/better-comments-next

**Comment by zxq08** [comment]
> try this: https://github.com/edwinhuish/better-comments-next试试这个：https://github.com/edwinhuish/better-comments-next

It's very useful, thanks a lot!
